src_python/yaafelib/audiofeature.py

```diff
@@ -25,14 +25,32 @@
 import yaafelib.core as yaafecore
 
```

Comment:
I checked all the print statements and they all had only 1 item in it. And as `print(a)` and `print a` is the same in python2, I thought the `__future__` is not required. I'll add it to avoid future troubles though :+1: 

If I'm adding `__future__` I'll also add `absolute_import` (from [docs](https://docs.python.org/2/library/__future__.html)) as that's also something that may cause confusion later.